### PR TITLE
Updating the model, removing "contacts" and replacing with "officials"

### PR DESCRIPTION
**Pull request by bkayser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,86 @@
...
+## Running the Application in Docker
 
-Docker Desktop
+The easiest way to run the application is to use Docker.
 
-## Getting started
+### Installation
 
-Enable your Google account for access:
+To install docker on a linux system, download the script at https://get.docker.com and run it:
```
Comment (saxond) [inline README.md:22]: I'd leave out linux instructions, but whatever.  It just seems like something that may or may not work depending on the distro, and will age poorly.

File: README.md [hunk 2]
```diff
@@ -1,62 +1,86 @@
...
+
+    docker compose up db
 
-### Running react app
+If you are running in production, you may need to override the `REACT_APP_API_URL` environment variable in the `app`
+container:
 
-From the app directory, run the initial setup:
+    REACT_APP_API_URL=https://api.ourofficials.com docker compose up
+
```
Comment (saxond) [inline README.md:62]: I'd use "in a docker container" instead of "out of docker"

File: README.md [hunk 3]
```diff
@@ -1,62 +1,86 @@
...
     npm install
 
+You will need to create a `.env` file in the `app` directory with the following contents:
+
+    REACT_APP_API_URL=http://localhost:8080
+
 Then run:
 
-    npm start
+    npm run start
```
Comment (saxond) [inline README.md:81]: `npm start` _does_ work

File: app/public/index.html [hunk 4]
```diff
@@ -2,21 +2,24 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <meta name="description"
+          content="Welcome to the OurOfficials.com! This site is dedicated to supporting sports official mentors."/>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;600&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap" rel="stylesheet">
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
+          rel="stylesheet">
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png"/>
```
Comment (saxond) [inline app/public/index.html:17]: Hmm..  Just wondering how `%PUBLIC_URL%` works when using a static build

**Review by saxond — APPROVED** [review]
Looks like code!

File: app/public/index.html (hunk 4 above)
Comment (bkayser) [reply app/public/index.html:17]: Dunno.  Was like that when I got there.

**Pull request merged**